src/components/search/SearchCourseCard.jsx

```diff
@@ -33,7 +35,7 @@ const SearchCourseCard = ({ hit, isLoading }) => {
 
   const partnerDetails = useMemo(
     () => {
-      if (!Object.keys(course).length) {
+      if (!Object.keys(course).length || !isDefinedAndNotNull(course.partners)) {
```

Comment:
Does one of our new utils cover the first part of this conditional now?

Reply:
Not currently. The first part of this conditional asks, "does this object contain any keys"? Our utils right now check for whether a value is `isDefined`, `null`, or evaluates to truthy, none of which get at that question.

Reply:
I can add a new util function, e.g. `isObjectAndHasKeys` or something like that if you think it'd be valuable?

Reply:
Nah, I don't think that's necessary. Just checking that what we currently had didn't check for an object with no keys